### PR TITLE
TESTS: expectRejected verifies the error msg

### PR DESCRIPTION
**Pull request by vistra**

### CODE REVIEW

File: test/protocol.spec.ts [hunk 1]
```diff
@@ -57,8 +57,8 @@ describe('protocol-contract', async () => {
 
     await evmIncreaseTimeForQueries(d.web3, 3);
 
-    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 3, {from: d.functionalOwner.address}));
-    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 2, {from: d.functionalOwner.address}));
+    await expectRejected(d.protocol.setProtocolVersion(DEPLOYMENT_SUBSET_MAIN, 3, currTime + 3, {from: d.functionalOwner.address}), /a protocol update can only be scheduled for the future/);
```
Comment (OdedWx) [inline test/protocol.spec.ts:60]: The set is done for currTime + 3, why is it in the future?

File: test/subscriptions.spec.ts [hunk 2]
```diff
@@ -133,25 +133,25 @@ describe('subscriptions-high-level-flows', async () => {
...
-    await expectRejected(d.subscriptions.removeSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), "Non-owner should not be able to remove a subscriber");
+    await expectRejected(d.subscriptions.removeSubscriber(subscriber.address, {from: d.contractsNonOwnerAddress}), /caller is not the functionalOwner/);
     r = await d.subscriptions.removeSubscriber(subscriber.address, {from: d.functionalOwner.address});
     expect(r).to.have.a.subscriberRemovedEvent({subscriber: subscriber.address})
   });
 
   it('should not add a subscriber with a zero address', async () => {
     const d = await Driver.new();
-    await expectRejected(d.subscriptions.addSubscriber(ZERO_ADDR, {from: d.functionalOwner.address}), "Should not deploy a subscriber with a zero address");
+    await expectRejected(d.subscriptions.addSubscriber(ZERO_ADDR, {from: d.functionalOwner.address}), /must provide a valid address/);
```
Comment (OdedWx) [inline test/subscriptions.spec.ts:154]: unnecessary check, in particular the check in the removeSubscriber

Comment (vistra) [reply test/subscriptions.spec.ts:154]: Right, removing

File: test/protocol.spec.ts (hunk 1 above)
Comment (vistra) [reply test/protocol.spec.ts:60]: `curTime` at this point is from about 3 seconds ago - see line 58. I'll make this clearer

**Pull request merged**